### PR TITLE
Mento Directory and QA Issue fix

### PR DESCRIPTION
**Pull request by vishwanath1004**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -32,11 +36,15 @@
     "@ionic/storage-angular": "^3.0.6",
     "@ngx-translate/core": "^13.0.0",
     "@ngx-translate/http-loader": "^6.0.0",
+    "ajv": "^6.12.6",
```
Comment (kiranharidas187) [inline package.json:39]: What is this plugin?

File: src/app/core/services/auth/auth.service.ts [hunk 2]
```diff
@@ -29,7 +31,8 @@ export class AuthService {
     try {
       await this.httpService.post(config);
       this.loaderService.stopLoader();
-      this.router.navigate([`/${CommonRoutes.AUTH}/${CommonRoutes.LOGIN}`],{replaceUrl:true});
+      this.toast.showToast('You have Signed up sucessfully, please use your registered id, password to login', 'sucess')
```
Comment (kiranharidas187) [inline src/app/core/services/auth/auth.service.ts:34]: translate

File: src/app/core/services/util/util.service.ts [hunk 3]
```diff
@@ -29,4 +34,77 @@ export class UtilService {
...
+              resolve(true);
+            }
+          }
+        ]
+      });
+      await alert.present();
+    });
+  }
+
+  getActionSheetButtons(type) {
```
Comment (kiranharidas187) [inline src/app/core/services/util/util.service.ts:68]: do we need this function here

File: src/app/pages/tabs/mentor-directory/mentor-directory.page.html [hunk 4]
```diff
@@ -1,20 +1,26 @@
...
+      </app-session-card>
     </div>
-  <div *ngFor="let value of mentor.values">
-  <app-session-card [data]="value"  [showAction]=true [action]="'VIEW_PROFILE'" (onClickEvent)="eventAction($event)"></app-session-card>
   </div>
+  <div *ngIf="!mentors?.length">
+    <app-no-data-found [message]="'NO_MENTORS_AVAILABLE'"> </app-no-data-found>
   </div>
 </ion-content>
+<ion-footer>
```
Comment (kiranharidas187) [inline src/app/pages/tabs/mentor-directory/mentor-directory.page.html:21]: Hide the loader if no more data

File: src/app/pages/tabs/mentor-directory/mentor-directory.page.ts [hunk 5]
```diff
@@ -1,64 +1,72 @@
...
+      this.loaderService.stopLoader();
+      this.mentors =  this.mentors.concat(data.result.data);
+      this.mentorsCount =  data.result.count;
+    }
+    catch (error) {
+      this.loaderService.stopLoader();
+    }
+  }
   eventAction(event) {
     console.log(event, "event");
```
Comment (kiranharidas187) [inline src/app/pages/tabs/mentor-directory/mentor-directory.page.ts:61]: rm all logs

File: src/app/shared/components/dynamic-form/dynamic-form.component.html [hunk 6]
```diff
@@ -93,7 +93,7 @@
     <small class="error-msg d-flex error-msg flex-align-items-center" >
       <div *ngIf="myForm.get(control.name).touched && myForm.get(control.name).invalid ">
         <ion-icon name="alert-circle-outline"></ion-icon>
-        <span class="mx-10">Invalid</span>
+        <span class="mx-10">Something is wrong</span>
```
Comment (kiranharidas187) [inline src/app/shared/components/dynamic-form/dynamic-form.component.html:96]: dont do this change. Revert it

File: src/app/shared/components/profile-image/profile-image.component.ts [hunk 7]
```diff
@@ -7,12 +8,34 @@ import { Component, Input, OnInit } from '@angular/core';
...
   @Input() showProfileDetails: any;
   @Input() username: any;
-  constructor() { }
+  @Input() uploadImage : boolean = false;
+  constructor(
+    private attachment : AttachmentService
+  ) { }
 
   ngOnInit() {
+    console.log(
```
Comment (kiranharidas187) [inline src/app/shared/components/profile-image/profile-image.component.ts:20]: rm

File: package.json (hunk 1 above)
Comment (vishwanath1004) [reply package.json:39]: while installing file-transfer plugin i was getting error message like ajv and ajv-keywords should be more than 6.*.* something so i added this

File: src/app/core/services/util/util.service.ts (hunk 3 above)
Comment (vishwanath1004) [reply src/app/core/services/util/util.service.ts:68]: Here am fetching action sheet buttons based on parameter

**Pull request merged**